core/node/eth_sender/src/eth_fees_oracle.rs

```diff
@@ -99,40 +115,37 @@ impl GasAdjusterFeesOracle {
         previous_sent_tx: &Option<TxHistory>,
         time_in_mempool_in_l1_blocks: u32,
     ) -> Result<EthFees, EthSenderError> {
+        const MIN_PRICE_BUMP_MULTIPLIER: f64 = 1.10;
+
         // we cap it to not allow nearly infinite values when a tx is stuck for a long time
         let capped_time_in_mempool_in_l1_blocks = min(
             time_in_mempool_in_l1_blocks,
             self.time_in_mempool_in_l1_blocks_cap,
```

Comment:
The time_in_mempool_cap will no longer work as intended after those changes, it has way too high default now + Without changes, I think it only makes sense for non-blob txs. The cap serves a very important purpose though, without it, it the eth-sender is stuck for a long time, it will start sending almost infinite values

Reply:
I think I can add a new config that will cap `param_b ^ time_in_mempool`. I'd use something like 10 as a default. I'll also check how it impacts the simulation I did before. IMO for blob txs it will still make sense (I'll have to add, currently it's not used there), can you elaborate why do you think it isn't?

Reply:
Resends are not limited. The formula is `a * b^t * median`, so it's 3 resends if median doesn't change and I think that's ok. If there is a spike median will increase